src/Language/SystemVerilog/Parser/Parse.y

```diff
@@ -721,27 +721,42 @@ AssignOption :: { AssignOption }
 -- for ModuleItem, for now
 AssertionItem :: { AssertionItem }
   : ConcurrentAssertionItem { $1 }
+  | DeferredImmediateAssertionItem { $1 }
 
 -- for Stmt, for now
 ProceduralAssertionStatement :: { Assertion }
   : ConcurrentAssertionStatement { $1 }
-  | ImmediateAssertionStatement  { $1 }
+  | SimpleImmediateAssertionStatement  { $1 }
+
+ImmediateAssertionStatement :: { Assertion }
+  : SimpleImmediateAssertionStatement  { $1 }
```

Comment:
I'd prefer we either keep the braces in this rule aligned, or not align them at all.